6_survival_analysis/nbconverted/survival_hit_groups.py

```diff
@@ -0,0 +1,704 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Survival analysis for Park et al. classes
+# 
+# For now, we'll just consider samples with a point mutation and CNV as "two-hit" samples, and samples with either but not both as "one-hit" samples. We could make this more granular in the future (i.e. use multiple point mutations as evidence of "two-hitness", or deep CNVs as "two-hit" samples as well).
+
+# In[1]:
+
+
+from pathlib import Path
+import pickle as pkl
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from sksurv.compare import compare_survival
+from sksurv.nonparametric import kaplan_meier_estimator
+from sksurv.util import Surv
+
+import sys; sys.path.append('..')
+import config as cfg
+
+
+# In[2]:
+
+
+MPMP_LOCATION = Path('/home/jake/research/mpmp')
+
+# park et al. geneset info
+park_loss_data = cfg.data_dir / 'park_loss_df.tsv'
+park_gain_data = cfg.data_dir / 'park_gain_df.tsv'
+
+# park et al. significant gene info
+park_loss_sig_data = cfg.data_dir / 'park_loss_df_sig_only.tsv'
+park_gain_sig_data = cfg.data_dir / 'park_gain_df_sig_only.tsv'
+
+MIN_N_MUTATED = 0
+MAX_MIN_PVAL = 0.05
+
+
+# ### Load information for Park et al. genes
+
+# In[3]:
+
+
+park_loss_df = pd.read_csv(park_loss_data, sep='\t', index_col=0)
+park_loss_df.head()
+
+
+# In[4]:
+
+
+park_gain_df = pd.read_csv(park_gain_data, sep='\t', index_col=0)
+park_gain_df.head()
+
+
+# In[5]:
+
+
+park_loss_sig_df = pd.read_csv(park_loss_sig_data, sep='\t', index_col=0)
+park_loss_sig_df.head()
+
+
+# In[6]:
+
+
+park_gain_sig_df = pd.read_csv(park_gain_sig_data, sep='\t', index_col=0)
+park_gain_sig_df.head()
+
+
+# ### Get clinical endpoint info
+# 
+# Code for preprocessing clinical info is based on: https://github.com/greenelab/mpmp/blob/master/mpmp/utilities/data_utilities.py#L510
+# 
+# TODO: move this to a function/utilities file
```

Comment:
Making a note of this in case you wanted to do this in this PR. No worries if you want to save for a later PR.

Reply:
I think I'll do this if we continue to use survival as an evaluation (e.g. in a future PR with gene expression data). Will keep it in mind.